packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -691,7 +690,37 @@ export function getPrismaClient(config: GetPrismaClientOptions) {
         finally(onFinally) {
           return request().finally(onFinally)
         },
+      } as Promise<unknown> & {
+        requestTransaction: (id: number) => Promise<unknown>
+      }
+    }
+
+    /**
+     * Executes a raw query provided through a safe tagged function
+     * @see https://github.com/prisma/prisma/issues/7142
+     *
+     * @param query
+     * @param values
+     * @returns
+     */
+    async $executeRaw(
+      query: TemplateStringsArray | sqlTemplateTag.Sql,
+      ...values: any[]
+    ) {
+      if (
+        (query as TemplateStringsArray).raw ||
+        (query as sqlTemplateTag.Sql).sql
+      ) {
+        return this.$executeRawRequest(query, ...values)
       }
+
+      throw new PrismaClientValidationError(`\`$executeRaw\` is a tagged function, please use it like the following:
```

Comment:
```suggestion
      throw new PrismaClientValidationError(`\`$executeRaw\` is a tag function, please use it like the following:
```

Looks like it's either a tag function or a tagged template from search online?

Reply:
If so needs to be done in other places as well

Reply:
Thanks!